mx.truffleruby/mx_truffleruby.py

```diff
@@ -19,7 +19,7 @@
     import mx_truffleruby_benchmark
 
 _suite = mx.suite('truffleruby')
-rubyDists = ['TRUFFLERUBY', 'TRUFFLERUBY-TEST']
+root = _suite.dir
```

Comment:
That will require to have Sulong as a sibling suite (mx restriction), i.e. in `../sulong`, is that OK?